brjs-core/src/main/java/org/bladerunnerjs/model/AbstractJsLib.java

```diff
@@ -42,6 +42,18 @@ public AbstractJsLib(RootNode rootNode, Node parent, MemoizedFile dir)
 	}
 	
 	@Override
+	public List<LinkedAsset> modelSeedAssets() 
```

Comment:
Fair enough, but `modelSeedAssets()` is a pre-existing abstract method on `AbstractBundlableNode`, and is nothing to do with this commit per-se. Maybe add a comment to the plug-in review issue?